node-export-server/server.js

```diff
@@ -441,7 +452,7 @@ function doExport(req, res, hash, input) {
                     if (req.query.crop) {
                         // if request did not provide a polygon or bounds to
                         // crop with, then go about zipping the output
-                        var cropShape = exports.validatePoly(req.query.poly) || exports.validateBbox(req.query.bbox);
+                        var cropShape = exports.polyQuotes(exports.validatePoly(req.query.poly)) || exports.validateBbox(req.query.bbox);
```

Comment:
I was seeing crashes when I passed the crop param and it appeared to be because the poly argument needed to have surrounding quotes